teams/leaderless.toml

```diff
@@ -0,0 +1,5 @@
+name = "leaderless"
+
+[people]
+leads = []
+members = ["pietroalbini"]
```

Comment:
Is this intentional or an artifact? It seems to not be used anywhere else...

Reply:
Woops, it was supposed to be inside the test suite.